deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages
+  apt: name={{ item }} state=present
+  with_items:
+    - postgresql-9.3
+    - postgresql-9.3-postgis-2.1
+    - libpq-dev
+    - python-dev
+    - python-psycopg2
+
+- name: Install pip packages
+  pip: name={{ item }}
+       virtualenv=/app_env
+  with_items:
```

Comment:
Yeah. We can put version numbers here but a requirements.txt is probably the cleaner way to do it.